lxd/instance/drivers/driver_qemu.go

```diff
@@ -3097,6 +3097,19 @@ func (d *qemu) deviceBootPriorities(base int) (map[string]int, error) {
 	return sortedDevs, nil
 }
 
+// shortenedSocketPath creates an alternative path to a socketTrickery by using the file descriptor to the directoryof the socket file.
```

Comment:
typo in the description `directoryof` also you mention > 107 chars here but earlier in this PR you also corrected a similar comment to > 108 chars, which is the correct one?

Reply:
`> 108` is the correct one, I think I followed the existing comment and later found that 108 is the actual limit

Reply:
I thought the tests caught typos in comments too, may be worth it to add this check in the future